packages/docmanager/test/dialog.spec.ts

```diff
@@ -70,3 +83,112 @@ describe('docregistry/dialog', () => {
     });
   });
 });
+
+/**
+ * A minimal docmanager dialog for test
+ */
+class MinimalCustomDialogs implements IDocumentManagerDialogs {
+  renameCalled = false;
+  confirmCloseCalled = false;
+  saveBeforeCloseCalled = false;
+
+  async rename(context: DocumentRegistry.Context): Promise<void | null> {
+    this.renameCalled = true;
+    return null;
+  }
+
+  async confirmClose(
+    args: IDocumentManagerDialogs.IConfirmCloseOptions
+  ): Promise<IDocumentManagerDialogs.IConfirmCloseResult> {
```

Comment:
A part of me wonders if we should nest the options and results in a namespace like `IDocumentManagerDialogs.ConfirmClose.IOptions` and `IDocumentManagerDialogs.ConfirmClose.IResult`. I am not sure.

Reply:
Working on making the change